VCAC-A/README.md

```diff
@@ -3,28 +3,29 @@ Intel VCAC-A is designed to accelerate analytics computation. This README descri
 
 ## 1. Setup VCAC-A:
 
-Please follow the [Software Installation Guide, Section 2](https://cdrdv2.intel.com/v1/dl/getContent/611894) to build and configure the software packages for the host and the VCAC-A, with the following additional steps:    
+Please follow the [Software Installation Guide, Section 2.0-Section 2.2.4](https://cdrdv2.intel.com/v1/dl/getContent/611894) to build and configure the software packages for the host and the VCAC-A, with the following additional steps:    
 
 - **Install Docker Engine on Host and VCAC-A**
 
-Follow the [instructions](https://docs.docker.com/v17.09/engine/installation) to install the latest docker engine on both the host and the VCAC-A. It is important that you properly setup proxies if you are behind a corporation firewall.    
+Follow the [instructions](https://docs.docker.com/v17.09/engine/installation) to install the latest docker engine on both the host and the VCAC-A. It is important that you properly setup proxies if you are behind a corporation firewall.
 
-- **Install Intel OpenVINO on VCAC-A**
+Here are the scripts to install Docker on the VCAC-A:
+-  **```Setup Passwordless access```** : [setup-access.sh](./script/setup-access.sh)
+-  **```Install Docker or Docker CE on VCAC-A```**: [setup-docker.sh](./script/setup-docker.sh)
+-  **```Install OpenVINO and start hddldaemon on VCAC-A```** : [setup-hddl.sh](./script/setup-hddl.sh)
 
-Login to VCAC-A and install the Intel OpenVINO software by downloading from this [link](https://software.intel.com/en-us/openvino-toolkit/choose-download). The package name should read ```l_openvino_toolkit_p_201x.x.xxx.tgz```. During installation, the only required component is ```Inference Engine```. 
-
-After installation, start the HDDL daemon as follows. 
+After installation, start the HDDL daemon manually as the following if hddldaemon does not start.
```

Comment:
... as follows ...

Reply:
Actually, this section is not needed as setup-hddl.sh will auto-start hddldaemon. The user should not care about hddldaemon.